### DISCUSSION TITLE
Focus issues with Radio Button Group

### DISCUSSION THREAD

**Discussion by jaymehta-twilio** [twilio-labs/paste#1782] [Q&A] [answered]
Hello, we've been using the radio button group in multiple places in our app. Since the radio buttons in the group carries the same `name` as attribute, the focus just jumps to the next focusable element as shown in the video. The focus doesn't go beyond the first radio button in that group in normal tab flow. Also, once an option is selected there's no way to go to another option. I'm not sure if this was intentional. Attaching the video for more clarity.

https://user-images.githubusercontent.com/52717786/129608154-08f20636-7d1a-4ea2-9b99-42367051d014.mov

**Comment by nkrantz** [comment, marked as answer]
Hi @jaymehta-twilio,

Thanks for starting this discussion. The tabbing interaction in your screen recording is actually expected behavior. Once you tab into the Radio Group, you can use up/down arrow keys to navigate between the Radio options.

You've brought up a great point, as this is a gap in our documentation. I've made [a ticket to add this info to the site](https://issues.corp.twilio.com/secure/RapidBoard.jspa?rapidView=748&projectKey=DSYS&view=planning&selectedIssue=DSYS-2994&issueLimit=100), feel free to follow along!

**Reply by jaymehta-twilio** [reply]
Thanks @nkrantz.
